pretixprint/app/src/full/java/eu/pretix/pretixprint/byteprotocols/LinkOSCard.kt

```diff
@@ -95,22 +95,64 @@ class LinkOSCard : CustomByteProtocol<Bitmap> {
             var zebraCardPrinter: ZebraCardPrinter? = null
 
             try {
-                var doubleSided = getSetting("hardware_${type}printer_doublesided", "false").toBoolean()
+                var forceDoubleSided = getSetting("hardware_${type}printer_doublesided", "false").toBoolean()
+                var canDoubleSided = false
                 val cardSource = getSetting("hardware_${type}printer_cardsource", "AutoDetect")
                 val cardDestination = getSetting("hardware_${type}printer_carddestination", "Eject")
 
                 zebraCardPrinter = ZebraCardPrinterFactory.getInstance(connection)
 
-                doubleSided = (zebraCardPrinter.printCapability == TransferType.DualSided && doubleSided)
+                canDoubleSided = zebraCardPrinter.printCapability == TransferType.DualSided
+                forceDoubleSided = (canDoubleSided && forceDoubleSided)
 
-                for (f in pages) {
+
+                var printJobs = mutableListOf<List<GraphicsInfo>>()
+                var pageoffset = 0
+                for (pagegroup in pagegroups) {
```

Comment:
I found this confusing on first read, because "for pagegroup in pagegroups" I'd expect that "pagegroup" holds an actual "group", such as, for example, the indexes of the grouped pages. But it does not, it more holds a "size of a pagegroup", so maybe "for groupsize in pagegroups" is clearer to read?